liquid-staking/src/delegation.rs

```diff
@@ -1,18 +1,23 @@
-use crate::{delegation_proxy, ERROR_BAD_WHITELIST_FEE};
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
 
-use super::errors::{
+use crate::basics::constants::{MIN_GAS_FINISH_EXEC, MIN_GAS_FOR_ASYNC_CALL, MIN_GAS_FOR_CALLBACK};
+use crate::{
+    basics::errors::{ERROR_BAD_WHITELIST_FEE, ERROR_INSUFFICIENT_GAS},
+    delegation_proxy,
+};
+
+use crate::basics::errors::{
     ERROR_ALREADY_WHITELISTED, ERROR_BAD_DELEGATION_ADDRESS, ERROR_CLAIM_EPOCH,
     ERROR_CLAIM_IN_PROGRESS, ERROR_DELEGATION_CAP, ERROR_FIRST_DELEGATION_NODE,
     ERROR_NOT_WHITELISTED, ERROR_NO_DELEGATION_CONTRACTS, ERROR_OLD_CLAIM_START,
     ERROR_ONLY_DELEGATION_ADMIN,
 };
-multiversx_sc::imports!();
-multiversx_sc::derive_imports!();
 
 pub const MAX_DELEGATION_ADDRESSES: usize = 20;
 pub const EGLD_TO_WHITELIST: u64 = 1_000_000_000_000_000_000;
 pub const MIN_BLOCKS_BEFORE_CLEAR_ONGOING_OP: u64 = 10;
```

Comment:
`pub const MIN_BLOCKS_BEFORE_CLEAR_ONGOING_OP: Blocks = 10;`

Reply:
also moved those to `constants.rs`